packages/procps/escape.c.patch

```diff
@@ -0,0 +1,52 @@
+--- procps-3.3.17.orig/proc/escape.c	2021-02-09 17:11:25.000000000 +0700
++++ procps-3.3.17/proc/escape.c	2021-05-12 00:30:51.000000000 +0700
+@@ -128,17 +128,18 @@
+   int my_bytes = 0;
+ 
+ #if (__GNU_LIBRARY__ >= 6) && (!defined(__UCLIBC__) || defined(__UCLIBC_HAS_WCHAR__))
+-  static int utf_init=0;
++  // This require android 26
```

Comment:
In most other [langinfo patches](https://github.com/search?q=org%3Atermux+nl_langinfo&type=code) we just replace `nl_langinfo(CODESET)` with `"UTF-8"` (since nl_langinfo is not available before api level 26). Is it any particular reason for commenting out everything here instead of just hardcoding UTF-8?

Reply:
Well i dont know that we can safety change it to "UTF-8", just try to patch to make it compile in this case,